src/h5web/visualizations/heatmap/DomainSlider.tsx

```diff
@@ -33,25 +33,34 @@ function DomainSlider(): JSX.Element {
   const step = Math.max((extendedMax - extendedMin) / 100, 10 ** -MIN_DECIMALS);
 
   return (
-    <ReactSlider
-      className={styles.slider}
-      trackClassName={styles.sliderTrack}
-      thumbClassName={styles.sliderThumb}
-      renderThumb={(props, state) => (
-        <div {...props}>{format(`.${MIN_DECIMALS}f`)(state.valueNow)}</div>
-      )}
-      value={[...(customDomain || dataDomain)]}
-      min={extendedMin}
-      max={extendedMax}
-      step={step}
-      pearling
-      onAfterChange={values => {
-        const roundedDomain = (values as number[]).map(value =>
-          round(value, MIN_DECIMALS)
-        ) as Domain;
-        setCustomDomain(roundedDomain);
-      }}
-    />
+    <div className={styles.sliderWrapper}>
+      <ReactSlider
+        className={styles.slider}
+        trackClassName={styles.sliderTrack}
+        thumbClassName={styles.sliderThumb}
+        renderThumb={(props, state) => (
+          <div {...props}>{format(`.${MIN_DECIMALS}f`)(state.valueNow)}</div>
+        )}
+        value={[...(customDomain || dataDomain)]}
+        min={extendedMin}
+        max={extendedMax}
+        step={step}
+        pearling
+        onAfterChange={values => {
+          const roundedDomain = (values as number[]).map(value =>
+            round(value, MIN_DECIMALS)
+          ) as Domain;
+          setCustomDomain(roundedDomain);
+        }}
+      />
+      <button
+        className={styles.resetButton}
+        type="button"
+        onClick={() => setCustomDomain(dataDomain)}
```

Comment:
You could disable the button if `customDomain` is `undefined`: `disabled={!customDomain}`

Reply:
Oh and it should be:

```jsx
onClick={() => setCustomDomain(undefined)}
```

When the custom domain is `undefined`, the visualization falls back to using the `dataDomain`. It allows doing stuff if no custom domain is set (like disabling this button).

Reply:
Clever :100:, I will do that !